src/pages/documentation/migrate-to-android.md

```diff
@@ -0,0 +1,89 @@
+| Campaign        | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaign.svg?logo=android&logoColor=white&label=campaign)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaign)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| CampaignClassic | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaignclassic.svg?logo=android&logoColor=white&label=campaignclassic)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaignclassic)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+
+### Manual library import
+
+If you are manually importing SDK libraries, ensure you identify all currently used Android 1.x libraries and switch them over to Android 2.0 libraries. The list of current Android 2.0 libraries can be found in the [current SDK versions document](sdk-versions.md#android) (in the Android section).
+
+### Gradle
+
+If you are using Gradle to manage your Adobe Experience Platform Mobile SDK dependencies, the following example shows you how to switch Android 1.x libraries to Android 2.0 libraries in your `build.gradle` file.
```

Comment:
If you are using Gradle to manage your app dependencies, the following example shows how to upgrade to latest version in the `build.gradle` file.